ctapipe_io_nectarcam/__init__.py

```diff
@@ -21,9 +24,62 @@
 from ctapipe.core import Provenance
 from astropy.io import fits
 from .containers import NectarCAMDataContainer
+from .constants import (
+    HIGH_GAIN, N_GAINS, N_PIXELS, N_SAMPLES
+)
 
 __all__ = ['NectarCAMEventSource']
 
+OPTICS = OpticsDescription(
+    # https://gitlab.cta-observatory.org/cta-science/simulations/simulation-model/verification/verification-process/mst-structure/-/blob/master/Appendix-MST-Structure.pdf
+    # version from 20 Jan 2022
+    'MST',
+    equivalent_focal_length=u.Quantity(16., u.m),
+    num_mirrors=1,
+    mirror_area=u.Quantity(106., u.m**2), #no shadowing, uncertainty is 0.5 m2
+    num_mirror_tiles=86, # Garczarczyk 2017
+)
+
+def load_camera_geometry(version=3):
+    ''' Load camera geometry from bundled resources of this repo '''
+    f = resource_filename(
+        'ctapipe_io_nectarcam', f'resources/NectarCam-{version:03d}.camgeom.fits.gz'
+    )
+    Provenance().add_input_file(f, role="CameraGeometry")
+    return CameraGeometry.from_table(f)
```

Comment:
Should set the frame of the CameraGeometry here to include the focal length.  This is needed for transforms later

Reply:
Done